sormas-backend/src/main/java/de/symeda/sormas/backend/user/UserFacadeEjb.java

```diff
@@ -737,7 +742,8 @@ public List<UserDto> getIndexList(UserCriteria userCriteria, Integer first, Inte
 
 		cq.select(user);
 
-		return QueryHelper.getResultList(em, cq, first, max, UserFacadeEjb::toDto);
+		final List<UserDto> resultList = QueryHelper.getResultList(em, cq, first, max, UserFacadeEjb::toDto);
+		return resultList.stream().distinct().collect(Collectors.toList());
```

Comment:
please add a comment why`.stream().distinct()` needed, and don't forget to create an issue for improvung this

Reply:
added a comment